activiti-api-process-model/src/main/java/org/activiti/api/process/model/payloads/TimerPayload.java

```diff
@@ -0,0 +1,245 @@
+package org.activiti.api.process.model.payloads;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.activiti.api.model.shared.Payload;
+
+public class TimerPayload implements Payload {
+
+    private String id;
+    
+    private Date dueDate;
+    private Date endDate;
+
+    private String executionId;
+    private boolean isExclusive;
+
+    private int retries;   
+    
+    private int maxIterations;
+    private String repeat;
+
+    protected String jobHandlerType;
+    protected String jobHandlerConfiguration;
+    
+    protected String exceptionMessage;
+
+    protected String tenantId;
+    protected String jobType; 
```

Comment:
Are all these fields used in the context of a timer(isExclusive, retries, maxIteration, repeat, jobHandlerType, jobHandlerConfiguration, exceptionMessage)? Let's expose only the ones that are used specifically for timers.

Reply:
Should be done: https://github.com/Activiti/activiti-api/pull/128/commits/c69f9dbef617105ba636704c4ba887bc78b3b2e1